gazebo_ros2_control_demos/package.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>gazebo_ros2_control_demos</name>
+  <version>0.0.1</version>
+  <description>gazebo_ros2_control_demos</description>
+
+  <maintainer email="ahcorde@osrfoundation.org">Alejandro Hernandez</maintainer>
+
+  <license>Apache License 2.0</license>
+
+  <url type="website">http://ros.org/wiki/gazebo_ros_control</url>
+  <url type="bugtracker">https://github.com/ros-simulation/gazebo_ros2_control/issues</url>
+  <url type="repository">https://github.com/ros-simulation/gazebo_ros2_control</url>
+
+  <author>Alejandro Hernández</author>
+
+  <buildtool_depend>ament_cmake</buildtool_depend>
+
+  <exec_depend>ament_index_python</exec_depend>
+  <exec_depend>gazebo_ros2_control</exec_depend>
```

Comment:
I think you need to add:

* `joint_trajectory_controller`
* `joint_state_controller`
* `hardware_interface`
* `transmission_interface`
* `robot_state_publisher`

Reply:
updated